Project/Assets/Prairie/Framework/Script/Twine/TwineNode.cs

```diff
@@ -52,152 +55,182 @@ public class TwineNode : MonoBehaviour
+                    this.isMinimized = false;
+                }
+                else {
+                    this.isMinimized = true;
+                }
+                if (this.isDecisionNode) {
+                    this.isOptionsGuiOpen = true;
+                } else if (this.isConditionNode) {
+                    // get the $color value from global list
+                    // check the platform name by check $color:platform pair stored in condition node
+                    // check child node name to match platform name
+                    // activate childnode
+                    this.ActivateChildAtIndex (0);
+                    Debug.Log ("This is a Condition Node: " + this.name);
+                }
+            }
         }
-        else if (this.enabled && !this.isMinimized) //Draw just this box
+    
+        private float vScrollBarValue;
+        public Vector2 scrollPosition = new Vector2(0, 0);
+        private string innerText;
+        public void OnGUI()
         {
-            float frameWidth = Math.Min(Screen.width / 3, 350);
-            float frameHeight = Math.Min(Screen.height / 2, 500);
-            Rect frame = new Rect(10, 10, frameWidth, frameHeight);
-            GUI.BeginGroup(frame);
-            GUIStyle style = new GUIStyle(GUI.skin.box);
-            style.wordWrap = true;
-            style.fixedWidth = frameWidth;
-            GUILayout.Box(this.content, style);
+            if ((fanfold) && (allMinimized == false)) {
+                float frameWidth = Math.Min(Screen.width / 5, 200);
+                float frameHeight = 80;
+                if (TwineNodeList.Count() > 0){
+                    frameHeight = Math.Min(Screen.height / (TwineNodeList.Count()), 80);
+                }
+                int index = TwineNodeList.IndexOf(this);
+                Rect frame = new Rect (10, 10+index*100, frameWidth, frameHeight);
+
+                GUIStyle style = new GUIStyle (GUI.skin.box);
+                style.wordWrap = true;
+                style.fixedWidth = frameWidth;
+
+                GUI.BeginGroup(new Rect(20, 10+index*frameHeight, frameWidth + 10, frameHeight));
+
+                scrollPosition = GUILayout.BeginScrollView(scrollPosition, GUILayout.Width(frameWidth + 10), GUILayout.Height(frameHeight));
+                content = content.TrimEnd();
+                GUILayout.Label(new GUIContent(this.content), style, GUILayout.Width(frameWidth - 50), GUILayout.ExpandHeight(true));
+                if (isDecisionNode) {
+                    GUIStyle decisionHintStyle = new GUIStyle (style);
+                    decisionHintStyle.fontStyle = FontStyle.BoldAndItalic;
+
+                    if (!isOptionsGuiOpen) {
+                        GUILayout.Box ("Press TAB to progress in the story...", decisionHintStyle);
+                    } else {
+                        GUILayout.Box ("Press TAB to scroll, E to close, ENTER to choose", decisionHintStyle);
```

Comment:
Are we sure we want "E" for both closing the option and carry? Is this conflict ever tested? 
Also didn't see where this behavior / keydown("E") is specified. Is it called in firstPersonInteractor?